tests/pyfunc/test_spark.py

```diff
@@ -267,7 +266,10 @@ def predict(self, context, model_input):
                 columns=["x", "b", "c", "d"], data={"x": [1], "b": [2], "c": [3], "d": [4]}
             )
         )
-        with pytest.raises(AnalysisException, match=r"cannot resolve 'a' given input columns"):
+        with pytest.raises(
+            AnalysisException,
+            match=r"cannot resolve 'a' given input columns|Column 'a' does not exist",
```

Comment:
New spark version error message changed to be `Column 'a' does not exist` . So I updated here.

Reply:
Compared with https://github.com/mlflow/mlflow/pull/6071, I preferred to support both old and new message format, so that developer running test over old spark version can also work.

Reply:
+1